5.0/apache/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM php:8-apache
+LABEL maintainer="markus@martialblog.de"
+ARG version='5.0.0+210526'
+ARG sha256_checksum='b82edc84970b438fdcc63880bb4dee74ee5afb61540f25be8c84a102881c2bc0'
+ARG USER=www-data
+ARG LISTEN_PORT=8080
+
+# Install OS dependencies
+RUN set -ex; \
+        apt-get update && \
+        DEBIAN_FRONTEND=noninteractive \
+        apt-get install --no-install-recommends -y \
+        \
+        libldap2-dev \
+        libfreetype6-dev \
+        libjpeg-dev \
+        libonig-dev \
+        zlib1g-dev \
+        libc-client-dev \
+        libkrb5-dev \
+        libpng-dev \
+        libpq-dev \
+        libzip-dev \
+        libtidy-dev \
+        libsodium-dev \
+        netcat \
+        curl \
+        \
+        && apt-get -y autoclean; apt-get -y autoremove; \
+        rm -rf /var/lib/apt/lists/*
+
+# Link LDAP library for PHP ldap extension
+RUN set -ex; \
+        ln -fs /usr/lib/x86_64-linux-gnu/libldap.so /usr/lib/
+
+# Install PHP Plugins and Configure PHP imap plugin
+RUN set -ex; \
+        docker-php-ext-configure gd --with-freetype=/usr/include/ --with-jpeg=/usr && \
+        docker-php-ext-configure imap --with-kerberos --with-imap-ssl && \
+        docker-php-ext-install -j5 \
+        exif \
+        gd \
+        imap \
+        ldap \
+        mbstring \
+        pdo \
+        pdo_mysql \
+        pdo_pgsql \
+        pgsql \
+        sodium \
+        tidy \
+        zip
+
+ENV LIMESURVEY_VERSION=$version
+
+# Apache configuration
+RUN a2enmod headers rewrite remoteip; \
+        {\
+        echo RemoteIPHeader X-Real-IP ;\
+        echo RemoteIPTrustedProxy 10.0.0.0/8 ;\
+        echo RemoteIPTrustedProxy 172.16.0.0/12 ;\
+        echo RemoteIPTrustedProxy 192.168.0.0/16 ;\
+        } > /etc/apache2/conf-available/remoteip.conf;\
+        a2enconf remoteip
+
+# Use the default production configuration
+RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
+
+# Download, unzip and chmod LimeSurvey from official GitHub repository
+RUN curl -sSL "https://github.com/LimeSurvey/LimeSurvey/archive/${version}.tar.gz" --output /tmp/limesurvey.tar.gz
```

Comment:
I think this will leave limesurvey.tar.gz in the image right?
Because you are storing it one layer and then deleting in the next.

Reply:
Yeah this seems odd... there's probably a better solution

Reply:
Just removed that extra layer. Thanks for the hint :1st_place_medal: